src/Environment.ts

```diff
@@ -51,6 +53,12 @@ export class CypherEnvironment {
         }
     }
 
+    public compile(element: CypherCompilable): string {
+        if (!isCypherCompilable(element)) throw new Error("Invalid element, missing `getCypher` method");
```

Comment:
What do you think about rephrasing this error message as the point is to avoid exposing `getCypher` to the users? Maybe something along the lines of "Element not compilable" and a link or something to the explanation of what is compilable.

Reply:
```suggestion
        if (!isCypherCompilable(element)) throw new Error("Can't compile. Passing a non Cypher Builder element to env.compile");
```